### DISCUSSION TITLE
Will RAGFlow supports Multi-Meta-RAG?

### DISCUSSION THREAD

**Discussion by kiddyu** [infiniflow/ragflow#2524] [Q&A] [answered]
知识库上传不同的产品说明书，当查询的时候需要根据产品的名称区分要查询的文档，避免查询A产品的信息返回的确实B产品的内容。这种能力请问ragflow当前支持么？

**Comment by KevinHuSh** [comment]
Yes, you could build an agent like this:
![image](https://github.com/user-attachments/assets/ae2154cf-05d5-4d48-aa16-4643170c26ae)

**Comment by aamir-gmail** [comment, marked as answer]
Can we integrate Agents into the chat interface, Right now the only way to run an agent is to do it on the agent page.

**Comment by KevinHuSh** [comment]
![image](https://github.com/user-attachments/assets/fd731aa2-6180-436e-94ea-57d26a802f2f)
